spec/models/content_page_spec.rb

```diff
@@ -43,4 +47,49 @@
     expect(saved_page.slug.count("-")).to be(3)
     expect(saved_page.title.count("-")).to be(2)
   end
+
+  # The order is a depth first search
+  context "Next and Previous pages" do
+    before(:each) do
+      ContentPage.destroy_all
```

Comment:
What happens when `ContentPage` is `nil` - It'd be worth getting a test block for that usecase.

Reply:
Also, I think this `before` block is not required because of line 8 in this file

Reply:
This is needed so that the earlier pages do not interfere with those created below to test Next and Previous

Really we should start using  the database cleaner gem in tests